tests/topotests/ldp-topo1/r1/ip_mpls_route.ref

```diff
@@ -3,3 +3,4 @@ xx as to xx via inet 10.0.1.2 dev r1-eth0 proto xx
 xx via inet 10.0.1.2 dev r1-eth0 proto xx
 xx via inet 10.0.1.2 dev r1-eth0 proto xx
 xx via inet 10.0.1.2 dev r1-eth0 proto xx
+
```

Comment:
Is this new line really necessary?

Also:
- the commit message associated to this change seems outdated now;
- please use a separate line for each sign-off.

Reply:
When I run ldp_topo1 test it fails without the extra line.   This is a run with the extra line removed.  ldp_topo1 test needs spaces after each line also.    I added the line back in and the test runs successfully

>               assert failures == 0, "MPLS table output for router r%s:\n%s" % (i, diff)
E               AssertionError: MPLS table output for router r1:
E                 --- actual MPLS table output
E                 +++ expected MPLS table output
E                 @@ -5,4 +5,3 @@
E                   XX             LDP   10.0.1.2  implicit-null   
E                   XX             LDP   10.0.1.2  implicit-null   
E                   XX             LDP   10.0.1.2  implicit-null   
E                 -
E               assert 1 == 0

Reply:
Hi Renato, we have addressed the following 2 issues.  (If more action is needed, please let us know.)  Thanks!

- the commit message associated to this change seems outdated now;
- please use a separate line for each sign-off.

Reply:
@lynne-volta that's strange. For me this topotest pass with or without this commit. The topotest code calls `rstrip()` (which removes newlines by default) in both the expected and obtained outputs, so it intrigues me that you got that error message (especially because you had to add a newline on r1's output only, while the other routers still lack a trailing newline).

Maybe your iproute2 version is too recent and topotest is having problems with that. Could you try running this test without your LDP OC changes and see if the problem still happens?

Reply:
@ksschoener the third commit message still has an extra indentation level and it doesn't match the actual code change (a newline that was added).

Reply:
Thanks @rwestphal.  I just fixed the third commit message (fixed spacing, updated comment to indicate what actually changed in the code).